docs/data/parquet_files.md

```diff
@@ -0,0 +1,87 @@
+
+The functionality for this is defined in the [EHR API](../../pixl_ehr/README.md), specifically in
+[`PIXLDatabase.get_radiology_reports`](../../pixl_ehr/src/pixl_ehr/_databases.py), which queries the
+<!-- TODO: add link to PIXL database documentation once we have that -->
+[PIXL database]() for the de-identified radiology reports of the current extract and collects them
+in a single _parquet_ file together with the `image_identifier` and `procedure_occurrence_id`.
+
+_This was implemented in [PR#203](https://github.com/UCLH-Foundry/PIXL/pull/203)_.
+
+## Exporting (copying from OMOP ES)
+
+As part of the PIXL pipeline, we copy the OMOP-ES public _parquet_ files  to an export directory, to
+prepare them for upload to the DSH. The export location is controlled by the `ParquetExport` class
+in [`exports.py`](../../pixl_core/src/core/exports.py).
+
+The final export directory will have the following structure:
+
+```
+exports
+└── <project_slug>
+    ├── all_extracts
+    │   └── <extract_datetime_slug>
+    │       ├── omop
+    │       │   └── public
+    │       │       └── PROCEDURE_OCCURRENCE.parquet
+    │       └── radiology
+    │           └── radiology.parquet
+    └── latest -> </symlink/to/latest/extract>
+```
+
+_This was implemented in [PR#182](https://github.com/UCLH-Foundry/PIXL/pull/182)._
+
+## Uploading to the DSH
+
+When an extract is ready to be published to the DSH, the PIXL pipeline will upload the **Public**
+and **Radiology** _parquet_ files to the `<project-slug>` directory where the DICOM datasets are
+stored (see below). The uploading is controlled by `upload_parquet_files` in
+[`upload.py`](../../pixl_core/src/core/upload.py) which takes a `ParquetExport` object as input to
+define where the _parquet_ files are located. `upload_parquet_files` is called by the
+`export-patient-data` API endpoint defined in the [EHR API](../../pixl_ehr/src/pixl_ehr/main.py),
+which in turn is called by the `extract_radiology_reports` command in the
+[PIXL CLI](../../cli/README.md).
+
+Once the parquet files have been uploaded to the DSH, the directory structure will look like this:
+
+```
+<project-slug>
+    ├── <extract_datetime_slug>
+    │   └── parquet
+    │       ├── PROCEDURE_OCCURRENCE.parquet
+    │       └── radiology.parquet
+    ├── <pseudonymised_ID_DICOM_dataset_1>.zip
+    └── <pseudonymised_ID_DICOM_dataset_2>.zip
+```
+
+_This was implemented in [PR#264](https://github.com/UCLH-Foundry/PIXL/pull/264)._
+
+## Testing
+
+Various _parquet_ files are provided throughout the repo to enable unit and system testing:
+
+- `cli/tests/resources/omop/` contains public and private parquet files together with an
+  `extract_summary.json` file to mimic the input received from OMOP-ES for the unit tests
+- `test/resources/omop/` contains public and private parquet files together with an
+  `extract_summary.json` file to mimic the input received from OMOP-ES for the system tests
```

Comment:
This is all changing in #274 . The parquet files are being split into batches and as Stef says it's being de-duped.